src/Dapr.AzureFunctions.Extension/Bindings/DaprPublishAsyncCollector.cs

```diff
@@ -27,10 +27,8 @@ public DaprPublishAsyncCollector(DaprPublishAttribute attr, DaprServiceClient da
 
         public Task AddAsync(DaprPubSubEvent item, CancellationToken cancellationToken = default)
         {
-            if (item.Topic == null)
-            {
-                item.Topic = this.attr.Topic ?? throw new ArgumentException("No topic information was found. Make sure it is configured either in the binding properties or in the data payload.", nameof(item));
-            }
+            item.Name ??= this.attr.Name ?? throw new ArgumentException("No pub/sub name was found. Make sure it is configured either in the binding properties or in the data payload.", nameof(item));
+            item.Topic ??= this.attr.Topic ?? throw new ArgumentException("No topic information was found. Make sure it is configured either in the binding properties or in the data payload.", nameof(item));
```

Comment:
I didn't know about `??=`. Fancy. :)

Reply:
Soon you'll be able to write the whole functions runtime in one big expression. C#13 maybe? 😆